query-engine/core-tests/tests/query_validation_tests/selection_contains_no_truthy_values.expected.json

```diff
@@ -20,6 +20,16 @@
           "name": "dob",
           "typeName": "DateTime",
           "isRelation": false
+        },
+        {
+          "name": "posts",
+          "typeName": "Post[]",
+          "isRelation": true
+        },
```

Comment:
This seems counter to the name of the test? unless truthy isn't referring to `isRelation`

Reply:
This test ensures that a query with a selection set that contains no truthy values returns an error. The error is EmptySelection. What you're looking at here is some metadata that's added to the error so that the client can render pretty errors. posts and count were added because I added a relation to the datamodel that's used for all these tests.